app/controllers/offers_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class OffersController < ApplicationController
+  before_action :authenticate_user!
+  before_action :require_active_subscription!
+
+  def new
+    @offer = Offer.new(conversation: @conversation)
+  end
+
+  def create
+    user_not_authorized unless conversation.business?(current_user)
+
+    @offer = Offer.new(**offer_params, conversation_id: params[:conversation_id])
```

Comment:
I had some troubles passing these params via `ActiveView` and path helpers

Reply:
How so?

Reply:
I expected `conversation_id` to be included in the `offer_params` out of the box as I used the `form_with model: [conversation, offer]` helper in `offers/_form.html.erb` view.

Reply:
I think for that to occur it has to be included in the `form` as a value. `offer_params` is requiring the value to be nested inside of `offer`, so the only way for `offer_params` to pass the `conversation_id` would be for the params to look like `offer[conversation_id]=...`. Since `conversation_id` is being supplied by the url param instead it won't be permitted by the `offer_params` method.

Reply:
If you add the conversation as a hidden input in the form that should do the trick - but that also feels like a bit of data coupling that might not be the best option.